staging/src/k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/fieldmanager.go

```diff
@@ -136,6 +137,9 @@ func (f *FieldManager) Apply(liveObj runtime.Object, patch []byte, force bool) (
 	apiVersion := fieldpath.APIVersion(f.groupVersion.String())
 	newObjTyped, managed, err := f.updater.Apply(liveObjTyped, patchObjTyped, apiVersion, managed, applyManager, force)
 	if err != nil {
+		if conflicts, ok := err.(merge.Conflicts); ok {
+			return nil, errors.NewApplyConflict(conflicts, err)
+		}
 		return nil, err
```

Comment:
Who's using this error? Because I suspect this one should be converted to an API error type eventually too. So I don't think it makes sense to be inconsistent here?

Reply:
Test added for this: https://github.com/kubernetes/kubernetes/pull/72578/files#diff-4ac5831d494b1b52c7c7be81e552a458R188